php/Terminus/Commands/SitesCommand.php

```diff
@@ -197,8 +197,8 @@ public function index($args, $assoc_args)
         }
         $sites = $this->sites->all();
 
-        if (count($sites) == 0) {
-            $this->log()->warning('You have no sites.');
+        if (empty($sites)) {
```

Comment:
Nope. I erased it by accident and had to put it back, so went with 'empty' this time.